hastory-data-gen/src/Data/Hastory/Gen.hs

```diff
@@ -9,10 +9,18 @@ import Data.GenValidity.Time ()
 import Data.Hastory
 import Test.QuickCheck
 
+instance Validity Entry
```

Comment:
The validity instances should be in the code, not in the tests.

Reply:
Will do. What's the rationale behind this? Is it because the instance of `Validity Entry` determines what is a valid `Entry`, and therefore is "business logic" while `GenValid` and `GenUnchecked` would not be declared were it not for tests?

Reply:
That's right. Validity is useful independent of tests